app/views/layouts/application.html.erb

```diff
@@ -51,6 +51,10 @@
         <%= javascript_include_tag "cable" %>
     <% end %>
 
+    <% if @is_revision %>
++      <meta name="robots" content="noindex">
++    <% end %>
+
```

Comment:
@RuthAtem Can you remove the plus signs? Thanks!

Also, `package.json` and `yarn.lock` are not needed in this PR.

Reply:
Hi @TildaDares Thanks for the observation. Please check if it's ok now